topics/computational-chemistry/tutorials/covid19-docking/tutorial.md

```diff
@@ -0,0 +1,342 @@
+>    - *"Minimum pH"*: `4.4`
+>    - *"Maximum pH"*: `10.4`
+> 2. Rename the output file `Enumerated candidates SMILES`.
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > The **Enumerate charges** {% icon tool %} tool is based on the Dimorphite-DL program {% cite Ropp2019 %}.
+>    {: .comment}
+>
+{: .hands_on}
+
+The output is another SMILES file, with several hundred entries.
+
+## Generate three-dimensional conformations
+
+So far our list of enumerated candidate compounds is still in SMILES format; we need to produce three-dimensional structures in SDF format for docking. This can be done with the **Compound conversion** {% icon tool %} tool.
+
+If you are not familiar with SMILES and SDF formats, consult the introductory [protein-ligand docking tutorial](../cheminformatics/tutorial.md) for more details.
+
+> ### {% icon hands_on %} Hands-on: Convert to SDF format
+>
+> 1. **Compound conversion** {% icon tool %} with the following parameters:
+>    - *"Molecular input file"*: `Enumerated candidates` dataset.
+>    - *"Output format"*: `MDL MOL format (sdf, mol)`
+>        - *"Generate 3D coordinates"*: `Yes`
+> 2. Rename the output file `Enumerated candidates SDF`.
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > The **Compound conversion** {% icon tool %} tool is based on the OpenBabel toolkit {% cite OBoyle2011 %}.
+>    {: .comment}
+>
+{: .hands_on}
+
+## Splitting the SD-file into a collection
+
+The next stage is to split the SD-file with the candidate ligands into a set of smaller SD-files.
+
+
+> ### {% icon question %} Questions
+>
+> Why is splitting the file necessary?
+>
+> > ### {% icon solution %} Solution
+> >
+> > The rDock tool performs one docking at a time (more technically: the task is not parallelized, as it uses only a single CPU). Therefore, splitting the large SD-file into many small files allows the work to be carried out by multiple Galaxy jobs in parallel, so it completes faster.
+> >
+> > In the original study, this kind of parallelization was absolutely essential because of the enormous dataset; at some points, there were 5,000 docking jobs running concurrently on the European Galaxy server. Even on the much smaller scale of this tutorial, we can speed things up considerably using this trick.
+> >
+> {: .solution}
+{: .question}
+
+> ### {% icon hands_on %} Hands-on: Split the SD-files
+>
+> 1. **Split file** {% icon tool %} with the following parameters:
+>    - *"Select the file type to split"*: `SD-files`
+>    - *"SD-file to split"*: `Enumerated candidates SDF`
+>        - *"Specify number of output files or number of records per file?"*: `Number of output files`
+>            - *"Number of new files"*: `10`
+>        - *"Method to allocate records to new files"*: `Alternate output files`
+>
+>
+{: .hands_on}
+
+***TODO***: *Consider adding a question to test the learners understanding of the previous exercise*
+
+
+# Active site preparation
```

Comment:
maybe move this one step up?

Reply:
I did it in the same order as on the covid19 website; we can change both I guess.

Reply:
Ah, true the different level. Yes this makes sense. Ok!